openmls/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "openmls"
 version = "0.3.0"
 authors = ["OpenMLS Authors"]
-edition = "2018"
+edition = "2021"
```

Comment:
Good to move it to 2021, but did you check everything that could be changed?

Reply:
Nope, I wanted the new resolver and this seemed like the easiest way. And nothing complained.
Note that the hpke version used here requires 2021 already. Nothing broke and cargo fix was happy as well.
I don't have any specific items we should change. But we should make use of `#![doc = include_str!("../Readme.md")]` for documentation.

Reply:
Maybe worthwhile creating a ticket to check if we should change anything.